testhelper/certificatetest.go

```diff
@@ -51,6 +53,23 @@ func GetRSALeafCertificate() RSACertTuple {
 	return rsaLeaf
 }
 
+// GetRevokableRSALeafCertificate returns leaf certificate that specifies a local OCSP server signed using RSA algorithm
+func GetRevokableRSALeafCertificate() RSACertTuple {
+	setupCertificates()
+	return revokableRSALeaf
+}
+
+// GetRevokableRSAChain returns a chain of certificates that specify a local OCSP server signed using RSA algorithm
+func GetRevokableRSAChain(size int) []RSACertTuple {
+	setupCertificates()
+	chain := make([]RSACertTuple, size)
+	chain[size-1] = rsaRoot
+	for i := size - 2; i >= 0; i-- {
+		chain[i] = getRevokableRSAChainCertTuple(fmt.Sprintf("Notation Test Revokable RSA Chain Cert %d", size-i), &chain[i+1], i)
+	}
```

Comment:
My thought process is that, in the future, the certs returned by this would have OCSPServer set as well as CRL Distribution Points. That way we don't need two separate functions for what are essentially just a slight difference in the template. That's why I gave it the more generic name of "revokable" rather than OCSP, since we can just specify different behaviors via the mocked client (as long as the endpoints are available on the certs)

Reply:
@priteshbandi does this explanation answer your question? If so, I can resolve this